src/pages/Settings/Settings.tsx

```diff
@@ -363,16 +365,35 @@ const WebauthnCredentialItem = ({
 	credential,
 	onDelete,
 	onRename,
+	unlocked
+
 }: {
 	credential: WebauthnCredential,
 	onDelete?: false | (() => void),
```

Comment:
This type annotation should be updated now that we've added an `await` on `onDelete()` below.
```suggestion
	onDelete?: false | (() => Promise<void>),
```

Reply:
Note also that `deleteWebauthnCredential` also calls `refreshData()` at the end, but without `await` - meaning that `await onDelete()` will _not_ wait for `refreshData()` to complete. If we want `onDelete()` to also wait for `refreshData()` to complete, we need to add an `await` to `refreshData()` as well (there are a few more occurrences like this that could use review too).